resources/views/entities/components/posts/children.blade.php

```diff
@@ -0,0 +1,62 @@
+
+
+@if($entity->hasChild())
```

Comment:
Remove all of this and make the datagrid init on page load to get rid of any sql query here in the view